cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java

```diff
@@ -10,11 +11,11 @@ public class AssignmentListDTO {
 	public static class AssignmentDTO {
```

Comment:
Should this class be `AssignmentListDTO`?

Reply:
Oops never mind, `public class AssignmentListDTO {` was hidden and I just had to expand it :sweat_smile:

Reply:
Awesome, glad it wasn't anything I had to fix. Thank you!